pages/common/git-sed.md

```diff
@@ -0,0 +1,22 @@
+# git sync
+
+> Run grep as directed but replace the given files with the pattern.
+> Works only on git tracked files.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-sed>.
+
+- Replace the string 'this' with 'that':
+
+`git sed 'this' 'that'`
```

Comment:
What if we use the placeholders from the `sed` page
```suggestion
`git sed '{{find_text}}' '{{replace_text}}'`
```

Reply:
Makes sense, thanks for the suggestion.